app/helpers/tasks_helper.rb

```diff
@@ -1,5 +1,5 @@
 module TasksHelper
 
   def tasks_path
-  end 
+  end
```

Comment:
This method isn't doing anything, so I'm not sure that you need it. Also, it would end up hiding the auto-generated method to the TaskController index page, if you had routes like that (see the comments on routing). I would just remove it for now.